### ISSUE TITLE
Change quiz & question format

### ISSUE THREAD

**Issue by Privat33r-dev** [play-with-docker/play-with-docker.github.io#258]
## Issue
Currently quiz looks and feels outdated.

<img src=https://github.com/play-with-docker/play-with-docker.github.io/assets/39376984/68bcc3fb-572f-47a7-9f4f-decfcb29db3d width=50%>

## Suggestions
 - Remove carousel, make questions one after another in a format similar to the doc formatting. 
 - Add open-ended questions.

**Comment by marcosnils** [comment]
> * Remove carousel, make questions one after another in a format similar to the doc formatting.
> * Add open-ended questions.

SGTM! I guess the follow-up for this issue is come up with a PoC about how this could look like. Have you seen any other ideas in similar websites / platforms? There's also https://labs.iximiuz.com/

**Comment by Privat33r-dev** [comment]
> I guess the follow-up for this issue is come up with a PoC about how this could look like.

Yep, but it's not that easy as it initially seemed. Prepared part of the design.

>Have you seen any other ideas in similar websites / platforms? There's also https://labs.iximiuz.com/

That's a good example, but I couldn't find quizzes there. I checked also edx, tryhackme, khan academy and a few others. I came up with some ideas, currently it's still in progress.

Nothing fancy so far. I am not a designer :)
![image](https://github.com/play-with-docker/play-with-docker.github.io/assets/39376984/93664572-8d02-4f3c-a333-38307f326435)

UPD: Redesigned :)
![image](https://github.com/play-with-docker/play-with-docker.github.io/assets/39376984/8f673174-f263-4e08-9831-d82a7cfa4433)
